lazer/contracts/sui/sources/pyth_lazer.move

```diff
@@ -31,13 +35,62 @@ fun init(_: PYTH_LAZER, ctx: &mut TxContext) {
     transfer::public_share_object(s);
 }
 
-/// Parse the Lazer update message and validate the signature.
+/// Verify LE ECDSA message signature against trusted signers.
+///
+/// This function recovers the public key from the signature and payload,
+/// then checks if the recovered public key is in the trusted signers list
+/// and has not expired.
 ///
+/// # Arguments
+/// * `s` - The pyth_lazer::state::State
+/// * `clock` - The sui::clock::Clock
+/// * `signature` - The ECDSA signature bytes (little endian)
+/// * `payload` - The message payload that was signed
+///
+/// # Errors
+/// * `ESignerNotTrusted` - The recovered public key is not in the trusted signers list
+/// * `ESignerExpired` - The signer's certificate has expired
+public fun verify_le_ecdsa_message(
```

Comment:
do we need to make it public :? maybe public in crate be enough (so we can test it, but users don't use it)